ERROR - 2022-07-03 13:36:46 --> Error connecting to the database: Unknown database 'sistema_inventario'
CRITICAL - 2022-07-03 13:36:46 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'sistema_inventario'
#0 C:\laragon\www\inventario_desarrollo-main\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\inventario_desarrollo-main\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\inventario_desarrollo-main\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\inventario_desarrollo-main\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\laragon\www\inventario_desarrollo-main\app\Controllers\SigninController.php(21): CodeIgniter\BaseModel->first()
#5 C:\laragon\www\inventario_desarrollo-main\system\CodeIgniter.php(832): App\Controllers\SigninController->loginAuth()
#6 C:\laragon\www\inventario_desarrollo-main\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#7 C:\laragon\www\inventario_desarrollo-main\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\inventario_desarrollo-main\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
